common/startup/check_local_php_exists.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
+function check_local_php_exists {
 # generate a local config file if it doesn't exist.
 # This is needed to ensure the db credentials can be prefilled in the UI, as env vars aren't taken into account.
-if [ ! -f $MAUTIC_VOLUME_CONFIG/local.php ]; then
-	cp -p /templates/local.php $MAUTIC_VOLUME_CONFIG/local.php
+if [ ! -f "${MAUTIC_VOLUME_CONFIG}/local.php" ]; then
+	cp -p /templates/local.php "${MAUTIC_VOLUME_CONFIG}/local.php"
+	echo "${MAUTIC_VOLUME_CONFIG}/local.php was not found, creating a fresh local.php from the template."
```

Comment:
```suggestion
	[[ $DEBUG -eq 1 ]] &&  echo "${MAUTIC_VOLUME_CONFIG}/local.php was not found, creating a fresh local.php from the template."
```

What do you think about printing it only when we require verbose output?

Reply:
question remains

Reply:
This feels more like a user needed info vs a debug message, to me.